pkg/users/account-create-dialog.js

```diff
@@ -167,31 +206,64 @@
     return remove_diacritics(result);
 }
 
-export function account_create_dialog(accounts) {
+export function account_create_dialog(accounts, shells) {
     let dlg = null;
     const state = {
+        dialogLoading: true,
         real_name: "",
         user_name: "",
         password: "",
         password_confirm: "",
         locked: false,
         confirm_weak: false,
         change_passw_force: false,
+        base_home_dir: null,
+        shell: null,
+        home_dir: null,
+        home_dir_dirty: false,
+        isShellSelectExpanded: false,
     };
     let errors = { };
 
     let old_password = null;
     let user_name_dirty = false;
 
+    function get_defaults() {
+        return cockpit.spawn(["useradd", "-D"], { superuser: "require", err: "message" })
+                .catch(e => console.warn("Could not get useradd defaults: ", e.message))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230314-140716-6b50d398-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)